### PR TITLE
Improve outline deployment

### PR DESCRIPTION
**Pull request by douglara**
### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v4/apps/outline.yml [hunk 1]
```diff
@@ -64,6 +64,10 @@ services:
             SMTP_TLS_CIPHERS: $$cap_smtp_tls_ciphers
             SMTP_SECURE: $$cap_smtp_secure
             DEFAULT_LANGUAGE: $$cap_default_language
+        caproverExtra:
+            - FROM outlinewiki/outline:$$cap_outline_version
+            - EXPOSE 3000
+            - CMD yarn db:migrate --env production-ssl-disabled; yarn start
```
Comment (githubsaturn) [inline public/v4/apps/outline.yml:70]: Is this tested? It looks like the command here is different from the snippet below?  `db:migrate` vs `sequelize:migrate` - are these both the same?

Comment (iamsid) [reply public/v4/apps/outline.yml:70]: https://github.com/outline/outline/issues/4772
It seems sequelize is no longer available.

**Comment by githubsaturn** [comment]
@iamsid - just to confirm your comment on the other thread, are you saying that even this PR does not work?

Should we just remove outline from the directory until it's fixed by someone?

**Comment by iamsid** [comment]
Did a bit more digging to identify the issue. And was able to install Outline.
Turns out, this merge-request/PR is not needed. But there are two issues with the existing YAML file.

Unfortunately, I don't know how to do a PR. So here are the two issues:
1. The pre-deploy script in current YAML file does not show correctly in the UI after installation is complete. Its marked using ``` this YAML code, but it shows incorrectly in CapRover's UI. The pre-deploy script is needed for the app to run the migrations and start working.
2. The 4th point in this section, called "Run DB migration", needs to be "Paste below pre-deploy script in App Config tab". And the pre-deploy script itself just needs to change `sequelize:migrate` to `db:migrate`.

File: public/v4/apps/outline.yml (hunk 1 above)
Comment (douglara) [reply public/v4/apps/outline.yml:70]: Maybe `sequelize:migrate` is deprecated command, I followed error instruction to fix

**Comment by douglara** [comment]
Sorry I created this PR with error

**Comment by douglara** [comment]
ixe

**Comment by douglara** [comment]
Sorry, I created this PR with error

Follow fixes 

@iamsid 
1 - I configured to run migration on application startup, now no needs step 4
2 - Now runs automatically

3 - Removed optional environment variables raising errors if not configured
4 - Add instruction to configure authentication method, outline works only thirdy party authentication methods

**Comment by githubsaturn** [comment]
Thanks!

**Pull request merged**